api/priceFeed.go

```diff
@@ -11,6 +15,7 @@ type PriceFeed interface {
 type FeedPair struct {
 	FeedA PriceFeed
 	FeedB PriceFeed
+	l     logger.Logger
```

Comment:
this looks like it's unused? -- better to combine it with the factory method that creates this otherwise it's a little harder to review :)

Reply:
there's a log call on line 35 that uses it; that does seem like a weird place to log though